common/core/desktop/src/kmx/kmx_processor.cpp

```diff
@@ -240,28 +249,43 @@ KMX_BOOL KMX_Processor::ProcessGroup(LPGROUP gp, KMX_BOOL *pOutputKeystroke)
         PKMX_WCHAR pdeletecontext = m_context.Buf(1);   // I4933
         if(!pdeletecontext || *pdeletecontext == 0) {   // I4933
           m_actions.QueueAction(QIT_INVALIDATECONTEXT, 0);
+          if(m_debug_items) {
+            m_debug_items->push_group_exit(gp, KM_KBP_DEBUG_FLAG_NOMATCH);
+          }
```

Comment:
No corresponding line in kmprocess.cpp. Should an edge case bug in the debugger!